.pre-commit-config.yaml

```diff
@@ -1,18 +1,23 @@
 repos:
-  - repo: git@github.com:syntaqx/git-hooks.git
+  - repo: https://github.com/syntaqx/git-hooks.git
```

Comment:
I assume these changes will not have any adverse effect as long as the pre-commit hooks we use are made publicly available, right? 

Also, did you test the changes locally? It'd be nice to have a check on the PR description if so.

Reply:
Yes to both questions. I think we can probably do something with the GitHub bot token to let us use SSH here, but we're extremely unlikely to use private pre-commit hooks (even if we had some of our own I'd expect them to be public, or defined within this repo)